using-mobile-extensions/adobe-journey-optimizer/api-reference.md

```diff
@@ -264,11 +281,7 @@ public static func setPushIdentifier(_ deviceToken: Data?)
 
 ## addPushTrackingDetails <a id="addpushtrackingdetails"></a>
 
-This API is used to updated the push notification pending intent with necessary customer journey information.
-
-{% hint style="warning" %}
-Call to this API is necessary to ensure that all the important information \(messageId, Customer journey information\) are added to the pending intent so that they can be used while tracking the push notification interactions.
-{% endhint %}
+The addPushTrackingDetails API is used to update a pending intent with important information, such as messageId and Customer Journey information. Please note that calling this API is mandatory, so the pending intent can be used while tracking push notification interactions.
```

Comment:
Should sentence two be an actual [!NOTE] ?

Reply:
GitBook doesn't support [!NOTE], so alas, I have to keep it like this. I can make it a bit clearer by bolding the note, though.